numpy/polynomial/_polybase.py

```diff
@@ -58,6 +58,11 @@ class ABCPolyBase(object):
     window : (2,) ndarray
         Default window of the class.
 
+    Notes
+    -----
+    ABCPolyBase instances should be immutable. Pleae ensure that any
+    modifications to this baseclass conform with this intention.
```

Comment:
Docstrings aren't really the place for notes aimed at people modifying the class - a consumer of the numpy docs does not care about that.

Did you mean subclasses, instead of modifications?

Reply:
I meant what I said, but thank you for informing me I was adding this not in a location that would target an inappropriate audience. I will correct, or remove, the note accordingly.

Reply:
Perhaps just put the word immutable on the first line of this docstring? `An abstract base class for immutable series classes`

Reply:
I prefer this approach. Thanks for the suggestion.